index.html

```diff
@@ -30,16 +46,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" style="background-color: teal;">Box 1</div>
+                <div class="box" style="background-color: gold">Box 2</div>
+                <div class="box" style="background-color: cadetblue">Box 3</div>
+                <div class="box" style="background-color: coral">Box 4</div>
+                <div class="box" style="background-color: crimson">Box 5</div>
+                <div class="box" style="background-color: forestgreen">Box 6</div>
+                <div class="box" style="background-color: darkorchid">Box 7</div>
+                <div class="box" style="background-color: hotpink">Box 8</div>
+                <div class="box" style="background-color: indigo">Box 9</div>
+                <div class="box" style="background-color: dodgerblue">Box 10</div>
             </div>
         
         </section>
```

Comment:
remember to link all navigation elements in both header and footer on all pages, you are linking them in the about page but not the footer links in the index page

Reply:
you can utilize the cursor:pointer property to indicate to the user that the buttons are to be interacted with and are clickable